src/Altinn.Notifications/Validators/SmsNotificationOrderRequestValidator.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Text.RegularExpressions;
+
+using Altinn.Notifications.Models;
+
+using FluentValidation;
+using PhoneNumbers;
+
+namespace Altinn.Notifications.Validators;
+
+/// <summary>
+/// Class containing validation logic for the <see cref="SmsNotificationOrderRequestExt"/> model
+/// </summary>
+public class SmsNotificationOrderRequestValidator : AbstractValidator<SmsNotificationOrderRequestExt>
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="SmsNotificationOrderRequestValidator"/> class.
+    /// </summary>
+    public SmsNotificationOrderRequestValidator()
+    {
+        RuleFor(order => order.Recipients)
+            .NotEmpty()
+            .WithMessage("One or more recipient is required.")
+            .Must(recipients => recipients.TrueForAll(a => IsValidMobileNumber(a.MobileNumber)))
+            .WithMessage("A valid mobile number must be provided for all recipients.");
```

Comment:
```suggestion
            .WithMessage("A valid mobile number istarting with country code must be provided for all recipients.");
```

Will result in a test or two needding to be updated as well